DESCRIPTION

```diff
@@ -32,7 +32,6 @@ Imports:
     glue,
     hardhat (>= 1.2.0),
     magrittr,
-    purrr,
```

Comment:
Is this just being removed b/c we implemented our own versions of the `purrr` functions?

Reply:
Yup. `purrr` is a huge package. So better to just keep the ones we need directly.